wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java

```diff
@@ -343,7 +343,7 @@ protected void loopFunc() {
 
     // Flush NetworkTables
     if (m_ntFlushEnabled) {
-      NetworkTableInstance.getDefault().flush();
+      NetworkTableInstance.getDefault().flushLocal();
```

Comment:
Pretty unlikely.  It’s just doing an async signal under the hood, which is a write()  to an async variable.  It’s better to have it be on by default because the alternative can result in unpredictable delays on writing values due to needing to resize a buffer, so having this is more deterministic.  It can still be disabled if users really care though.